mir-module/src/main/resources/META-INF/resources/content/publish/index.xml

```diff
@@ -74,7 +74,7 @@
       <div id="more_options_box" class="row form-group">
         <label for="genre" class="col-sm-3 col-lg-2 control-label">mit der PPN:</label>
         <div class="col-sm-7 col-md-5  col-lg-4">
-          <input type="text" name="ppn" id="ppn" class="form-control" size="20" placeholder="z.B. 198562268" />
+          <input type="text" name="ppn" id="ppn" class="form-control" size="20" placeholder="e.g. 198562268" />
         </div>
```

Comment:
This should also be translated completely and so seperated in <section xml:lang="de"> and <section xml:lang="en">